x/amm/migrations/v13_migration.go

```diff
@@ -0,0 +1,14 @@
+package migrations
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+func (m Migrator) V13Migration(ctx sdk.Context) error {
+	legacyParams := m.keeper.GetLegacyParams(ctx)
```

Comment:
no need for legacy params, Get Params, then if nothing has to be set, then can leave empty